internal/cli/deployment_list.go

```diff
@@ -267,16 +276,30 @@ func (c *DeploymentListCommand) Run(args []string) int {
 
 			sort.Strings(details)
 
+			var columns []string
+
+			columns = []string{
+				status,
+				c.flagId.FormatId(b.Sequence, b.Id),
+				b.Component.Name,
+				details[0],
+				startTime,
+				completeTime,
+				statusReportComplete,
+			}
+
+			if c.flagUrl {
+				theUrl := "N/A"
+				if b.Url != "" {
+					theUrl = b.Url
+				} else if b.Preload.DeployUrl != "" {
+					theUrl = b.Url
+				}
+				columns = append(columns, theUrl)
```

Comment:
quick bug fix and a couple of syntactical preferences to match the repo's style
```suggestion
				url := "n/a"
				if b.Url != "" {
					url = b.Url
				} else if b.Preload.DeployUrl != "" {
					url = b.Preload.DeployUrl
				}
				columns = append(columns, url)
```

Reply:
Sounds good! I tend not to use `url` because it conflicts with the `net.url` package, but since we're not using it here all good!